packages/blocks/README.md

```diff
@@ -660,6 +660,15 @@ _Returns_
 
 -   `?WPBlock`: The block, if it has been successfully registered; otherwise `undefined`.
 
+<a name="registerBlockVariation" href="#registerBlockVariation">#</a> **registerBlockVariation**
+
+Registers a new block variation for the given block.
```

Comment:
… for the given block **type**? block **name**?

Reply:
I would say the block type, the name is just a detail here.